openmp/runtime/test/ompt/callback.h

```diff
@@ -190,8 +190,8 @@ ompt_label_##id:
 // On AArch64 the NOP instruction is 4 bytes long, can be followed by inserted
 // store instruction (another 4 bytes long).
 #define print_possible_return_addresses(addr) \
-  printf("%" PRIu64 ": current_address=%p or %p\n", ompt_get_thread_data()->value, \
-         ((char *)addr) - 4, ((char *)addr) - 8)
+  printf("%" PRIu64 ": current_address=%p or %p or %p\n", ompt_get_thread_data()->value, \
```

Comment:
We probably want to limit this to macos only, so it might be a good idea to guard it with proper macros.